package.json

```diff
@@ -24,6 +24,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
     "react-spring": "^8.0.27",
+    "react-springs": "^1.1.0",
```

Comment:
Maybe this was removed later, but is this an actual package? It looks like it might be a typo for `react-spring`.

Reply:
Oops yeah this was definitely a typo. Will remove it